### DISCUSSION TITLE
Parallel Jobs and Workflows

### DISCUSSION THREAD

**Discussion by avinash371** [community/community#39737] [Actions]
### Select Topic Area

Question

### Body

Hi everyone,

How can we run parallel jobs in a workflow which is consisting pipeline?
And how can we run parallel workflows of multiple repositories at a time ? 
Consider there are self-hosted Runners for the organization which has 1500 repositories.

Regards,
Avinash

**Comment by airtower-luna** [comment]
Jobs will automatically run in parallel as long as there are enough runners available and there is no [`needs` relationship](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idneeds) preventing it (only possible within a workflow). Jobs that can't get a runner immediately will queue.

**Reply by avinash371** [reply]
How do workflows of different repositories run in parallel, not in a Queue?

**Reply by airtower-luna** [reply]
They will always run in parallel *as long as you have enough runners*. Each runner handles one job at a time. For GitHub-hosted runners [there are documented limits](https://docs.github.com/en/actions/learn-github-actions/usage-limits-billing-and-administration#usage-limits), for your self-hosted runners the limit is the number of runners you have.
